src/cloudant/_common_util.py

```diff
@@ -234,6 +234,24 @@ def codify(code_or_str):
         return _Code(code_or_str)
     return code_or_str
 
+def get_docs(url, **kwargs):
+    """
+    Provides a helper to get a response with documents for GET/POST requests.
+    """
+    r_session = kwargs.pop('r_session', None)
+    encoder = kwargs.pop('encoder', None)
+
+    params = python_to_couch(kwargs)
```

Comment:
This function will not work without a valid session object, that being the case why not specify it in the function signature as a mandatory argument that way you would get an error straight away if there is no session provided?  Also I feel like the `encoder` should explicitly be called out in the signature as well rather than including it as part of **kwargs.  `kwargs` also is not very expressive as to what they are.  My suggestion would be to change lines 237-244 above to:

```python
def get_docs(r_session, url, encoder=None, **params):
    """
    Provides a helper to get a response with documents for GET/POST requests.
    """
    params = python_to_couch(params)
```